src/ProductBundle/Command/ImportMassProductCommand.php

```diff
@@ -0,0 +1,426 @@
+<?php
+
+/*
+ * This file is part of the Sonata package.
+ *
+ * (c) Rascar Sylvain <sylvain.rascar@fullsix.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ProductBundle\Command;
+
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Util\Inflector;
+use Doctrine\ORM\EntityManager;
+use Psr\Log\LoggerInterface;
+use Sonata\Component\Product\ProductInterface;
+use Sonata\Component\Product\ProductManagerInterface;
+use Sonata\MediaBundle\Entity\MediaManager;
+use Sonata\MediaBundle\Model\MediaInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Application\Sonata\MediaBundle\Entity\Media;
+
+class ImportMassProductCommand extends ContainerAwareCommand
+{
+    /**
+     * @var boolean
+     */
+    protected $verbose;
```

Comment:
This is not used. Moreover, the Console component already provides a verbosity level.